hedera-node/src/main/java/com/hedera/services/store/tokens/ExceptionalTokenStore.java

```diff
@@ -131,27 +136,27 @@ public boolean isCreationPending() {
 	}
 
 	@Override
-	public void setAccountsLedger(TransactionalLedger<AccountID, AccountProperty, MerkleAccount> accountsLedger) {
+	public void setAccountsLedger(final TransactionalLedger<AccountID, AccountProperty, MerkleAccount> accountsLedger) {
 		/* No-op */
 	}
 
 	@Override
-	public void setHederaLedger(HederaLedger ledger) {
+	public void setHederaLedger(final HederaLedger ledger) {
 		/* No-op */
 	}
 
 	@Override
-	public void apply(TokenID id, Consumer<MerkleToken> change) {
+	public void apply(final TokenID id, final Consumer<MerkleToken> change) {
 		throw new UnsupportedOperationException();
 	}
 
 	@Override
-	public boolean exists(TokenID id) {
+	public boolean exists(final TokenID id) {
 		throw new UnsupportedOperationException();
 	}
 
 	@Override
-	public MerkleToken get(TokenID id) {
+	public MerkleToken get(final TokenID id) {
 		throw new UnsupportedOperationException();
 	}
 }
```

Comment:
Let's revert those changes - `final` usages.

Reply:
I'm unresolving this for now, as `final` was applied to all methods in the class.